.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+### Description
+Please explain the changes you made here.
+
+### Checklist
+- [ ] Code compiles correctly
```

Comment:
I'd also add the following:
* Commits are GPG-signed
* Commits contain the `Signed-off-by:` clause
* Commit messages are formatted according to https://www.conventionalcommits.org/en/v1.0.0/
* Commits are rebased on current master